src/util/graph.h

```diff
@@ -461,12 +464,21 @@ void grapht<N>::visit_reachable(node_indext src)
 template<class N>
 std::vector<typename N::node_indext>
 grapht<N>::get_reachable(node_indext src, bool forwards) const
+{
+  std::vector<node_indext> src_vector;
+  src_vector.push_back(src);
```

Comment:
I would suggest `std::vector<node_indext> src_vector({src})`

Reply:
Hmm, so appveyor did not like this, so I reverted to the original version.